store/src/java/com/zimbra/soap/ZimbraSoapContext.java

```diff
@@ -268,8 +268,11 @@ public ZimbraSoapContext(ZimbraSoapContext zsc, AuthToken authToken, String targ
     public ZimbraSoapContext(Element ctxt, QName requestName, DocumentHandler handler, Map<String, Object> context,
             SoapProtocol requestProtocol) throws ServiceException {
 
-        if (ctxt != null && !ctxt.getQName().equals(HeaderConstants.CONTEXT))
-            throw new IllegalArgumentException("expected ctxt, got: " + ctxt.getQualifiedName());
+        if (ctxt != null && !ctxt.getQName().equals(HeaderConstants.CONTEXT)) {
+            IllegalArgumentException e = new IllegalArgumentException("expected ctxt, got: " + ctxt.getQualifiedName());
```

Comment:
Why not just `sLog.warnQuietlyFmt("Illegal context name - expected 'ctxt', got: '%s'", ctxt.getQualifiedName");`